WordPress/src/main/java/org/wordpress/android/ui/media/MediaItemFragment.java

```diff
@@ -416,9 +414,4 @@ private void copyUrlToClipboard() {
             ToastUtils.showToast(getActivity(), R.string.error_copy_to_clipboard);
         }
     }
-
-    @Subscribe(threadMode = ThreadMode.MAIN)
-    public void onMediaChanged(MediaStore.OnMediaChanged event) {
```

Comment:
This crash if we don't remove `mDispatcher.register()` / `mDispatcher.unregister()` calls, and we can also remove `mDispatcher` altogether.